src/plugins/console/server/lib/spec_definitions/json/generated/snapshot.create.json

```diff
@@ -11,6 +11,6 @@
     "patterns": [
       "_snapshot/{repository}/{snapshot}"
     ],
-    "documentation": "https://www.opensearch.org/guide/en/elasticsearch/reference/master/modules-snapshots.html"
+    "documentation": "https://opensearch.org/docs/latest/opensearch/snapshot-restore/"
```

Comment:
Should this be https://opensearch.org/docs/1.2/opensearch/snapshot-restore/#take-snapshots?